frontend_v2/src/app/components/challenge/challengephases/editphasemodal/editphasemodal.component.html

```diff
@@ -135,6 +136,43 @@
         </div>
       </div>
 
+      <div class="col-lg-4 col-md-4 col-sm-12 col-lr-pad">
+        <div class="input-field align-left">
+          <span class="text-med-black fw-regular fs-16">Max Concurrent Submissions Allowed</span>
+          <app-input
+            class="mt-20"
+            [editPhaseDetails]="editPhaseDetails"
+            [name]="'max_concurrent_submissions_allowed'"
+            [isRequired]="true"
+            [label]="'max_concurrent_submissions_allowed'"
+            [placeholder]="''"
+            [type]="'number'"
+            [value]="maxConcurrentSubmissionsAllowed"
+            #formmodal
+          ></app-input>
+        </div>
+      </div>
+    </div>
+      
+      <div class="row row-lr-margin">
+        <div class="col-lg-6 col-md-6 col-sm-6 col-lr-pad">
+          <div class="input-field align-left">
+            <div class="fs-16 fw-regular text-med-black">Allowed Submission File Types</div>
```

Comment:
Can we please keep `Max Concurrent Submissions Allowed` and `Allowed Submission File Types` in the same row as there is a lot of empty space on the right side. Please see the ss below -
<img width="796" alt="Screenshot 2021-06-29 at 9 04 22 PM" src="https://user-images.githubusercontent.com/44888949/123826851-c22dea00-d91d-11eb-8492-a57497797cd6.png">

Reply:
Done!